src/db/schema.ts

```diff
@@ -7,6 +7,8 @@ export const usersTable = pgTable("users", {
   passwordHash: text("password_hash").notNull(),
   avatarUrl: text("avatar_url"),
   createdAt: timestamp("created_at").notNull().defaultNow(),
+  verificationToken: text("verification_token").unique(),
+  verificationTokenExpiry: timestamp("verification_token_expiry"),
```

Comment:
Don't add `verificationToken` and `verificationTokenExpiry` attributes to user table. Use the `token` table for verification